bin/mb

```diff
@@ -379,6 +380,7 @@ var aliases = {
         logfile: options.logfile,
         loglevel: options.loglevel,
         allowInjection: options.allowInjection,
+        savefile: options.savefile,
```

Comment:
This is right, but the default value is tripping you up (roughly line 354). The fact that it defaults to 'mb.json' means if you start it up without the --savefile option, mb is automatically setting it, which is why you think you need the --saveImposters flag. If you remove the default you can rely on --savefile entirely. You'll have to add the default for 'mb save' in the appropriate section (roughly line 431):

````
.command('save', function (saveYargs) {
  saveYargs.options({
    savefile: options.savefile || 'mb.json'
  });
}
````